src/sentry/release_health/metrics_sessions_v2.py

```diff
@@ -45,6 +60,8 @@
 
 logger = logging.getLogger(__name__)
 
+Scalar = Union[int, float, None]
```

Comment:
This just fits my mental model, which is

```
        ┌─ int
        │
 Scalar─┼─ float
        │
        └─ None
```
rather than
```
         ┌── int
       ┌─┤
 Scalar│ └── float
       └─ None

Reply:
Yep this was a personal nit-pick. In fact, at some point lower in the code it was probably going to generate confusion, or make the code less readable.